app/experimenter/experiments/tests/api/v5/test_query.py

```diff
@@ -79,3 +79,54 @@ def test_experiment_by_slug_not_found(self):
         content = json.loads(response.content)
         experiment = content["data"]["experimentBySlug"]
         self.assertIsNone(experiment)
+
+    def test_experiments_by_status(self):
+        user_email = "user@example.com"
+        draft_exp = NimbusExperimentFactory.create_with_status(
+            NimbusExperiment.Status.DRAFT
+        )
+        accepted_exp = NimbusExperimentFactory.create_with_status(
+            NimbusExperiment.Status.ACCEPTED
+        )
+
+        response = self.query(
+            """
+            query {
+                experimentsByStatus(status: DRAFT) {
+                    name
+                    slug
+                    publicDescription
+                }
+            }
+            """,
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200)
+        content = json.loads(response.content)
+        experiments = content["data"]["experimentsByStatus"]
+        self.assertEqual(len(experiments), 1)
+        for key in experiments[0]:
+            self.assertEqual(
+                experiments[0][key], str(getattr(draft_exp, to_snake_case(key)))
+            )
+
+        response = self.query(
+            """
+            query {
+                experimentsByStatus(status: ACCEPTED) {
+                    name
+                    slug
+                    publicDescription
+                }
+            }
+            """,
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200)
+        content = json.loads(response.content)
+        experiments = content["data"]["experimentsByStatus"]
+        self.assertEqual(len(experiments), 1)
+        for key in experiments[0]:
+            self.assertEqual(
+                experiments[0][key], str(getattr(accepted_exp, to_snake_case(key)))
+            )
```

Comment:
I don't think this is really necessary here?  Should be sufficient to just test against a single status, shouldn't it?